src/legacy/audio_ail.go

```diff
@@ -338,7 +335,7 @@ func Nox_xxx_WorkerHurt_44D810() int32 {
 	C.dword_587000_122848 = C.uint32_t(bool2int(C.dword_5d4594_831092 != 0))
 	ptr_830876 := memmap.PtrOff(0x5D4594, 830876)
```

Comment:
```suggestion
	ptr_830876 := (*C.timer)(memmap.PtrOff(0x5D4594, 830876))
```
Maybe assign type earlier? This way we can propagate it to `Sub_4864A0` as well.